Makefile

```diff
@@ -10,4 +10,4 @@ push: image
 		docker push ${IMAGE}:${ARCH}
 
 manifest: push
-		manifest-tool push from-args --platforms linux/amd64,linux/arm,linux/arm64 --template ${IMAGE}:ARCH --target ${IMAGE}
+		manifest-tool push from-args --platforms linux/arm64 --template ${IMAGE}:ARCH --target ${IMAGE}
```

Comment:
It seems some undesired changes have sneaked in your PR, like everything in this file for example

Reply:
Indeed, I messed up: additional commits made their way in the PR